### ISSUE TITLE
📋 TASK 4: ALL TEAM - WEEKLY SYNC & INTEGRATION (DEADLINE: 30/4)

### ISSUE THREAD

**Issue by Minh08012005** [Minh08012005/Heart-Disease-Prediction-ML-Group-6#4]
### TASK-04: Weekly Sync & Integration Check

**Assignees:** @Minh, @Tuân, @Hiếu, @Phong  
**Deadline:** 30/4/2026  
**Priority:** MEDIUM 🟡  
**Labels:** meeting, integration, review  
**Status:** To Do  

**📅 MEETING SCHEDULE:**
- **26/4 (Thứ 7) - 19:00**: Checkpoint 1 (Minh demo preprocessing xong)
- **30/4 (Thứ 4) - 19:00**: Final Review (Demo algorithms)

**🎯 MỤC TIÊU BUỔI 30/4:**
Tất cả code tích hợp được vào `main.py` và chạy được end-to-end.

**📝 CÔNG VIỆC CHUNG:**
1. **Minh**: Tạo `main.py` skeleton
   ```python
   # main.py
   from src.models.naive_bayes import NaiveBayes
   from src.models.decision_tree import DecisionTree
   from src.utils import train_test_split, accuracy_score
   
   # Load data
   # Preprocessing
   # Train/test split
   # Train models
   # Evaluate


2. __Tất cả__: Integration testing

   - Đảm bảo imports work
   - Đảm bảo API consistency (fit/predict)

__✅ DELIVERABLES (30/4):__

- `main.py` chạy được end-to-end
- Cả 2 models train được
- Có evaluation metrics cơ bản
- Meeting notes & action items cho tuần sau

__📊 SUCCESS CRITERIA:__

- Project chạy được: `python main.py`
- Không có lỗi import
- Có output metrics (accuracy)

```
```
